borgmatic/commands/arguments.py

```diff
@@ -66,7 +73,18 @@ def parse_subparser_arguments(unparsed_arguments, subparsers):
                     if item in subparsers:
                         remaining_arguments.remove(item)
 
-        arguments[canonical_name] = parsed
+        if canonical_name not in subcommand_parsers_mapping:
```

Comment:
You might consider removing the `not` and reversing the logic. Or even:

```python
arguments[canonical_name] = None if canonical_name in subcommand_parsers_mapping else parsed
```

Up to you!